package.json

```diff
@@ -105,6 +105,7 @@
     "pegjs-loader": "^0.5.6",
     "platformicons": "^4.3.0",
     "po-catalog-loader": "2.0.0",
+    "prettier": "^2.6.0",
```

Comment:
Intentional?

Reply:
Yeah, because I want the output of the script to reflect what is in the source tree so that when folks run devserver, it has no conflicts with the source. Right now we only install prod deps when building.

Reply:
@JonasBa 
>Right now we only install prod deps when building.

Are you sure about that? Because we need the dev deps to be able to build AFAIK?

Reply:
Scratch that, found the line I wrote: https://github.com/getsentry/sentry/blob/36341dff1c6e20946889c82aa2657cd51bccaf9b/docker/builder.dockerfile#L39

Don't remember a great reason for this so you may try removing that `--production` flag and see how bad this gets 😅

Reply:
Okay so that line is to mimic what we do in https://github.com/getsentry/sentry/blob/36341dff1c6e20946889c82aa2657cd51bccaf9b/src/sentry/utils/distutils/commands/base.py#L129

And that `--production` decision was made very very long ago: https://github.com/getsentry/sentry/commit/632c8f336ebaacf60221706e78d7881fe69f2251

Pretty sure it can be revisited.